client/src/config.js

```diff
@@ -0,0 +1,2 @@
+const GoogleLoginUrl = 'http://localhost:8000/auth/google';
```

Comment:
Please get this value from the envs so that you can configure it for production env.